hera_sim/tests/test_zernikebeam.py

```diff
@@ -0,0 +1,148 @@
+import numpy as np
+from hera_sim.visibilities import VisCPU
+from hera_sim import io
+from hera_sim.beams import ZernikeBeam
+from hera_sim.defaults import defaults
+
+np.seterr(invalid="ignore")
+
+
+def antennas():
+    locs = [[308, 253, 0.49], [8, 299, 0.22]]
+
+    ants = {}
+    for i in range(len(locs)):
+        ants[i] = (locs[i][0], locs[i][1], locs[i][2])
+
+    return ants
+
+
+def sources():
+    sources = np.array(
+        [
+            [
+                128,
+                -29,
+                4,
+                0,
+            ]
+        ]
+    )
+    ra_dec = sources[:, :2]
+    flux = sources[:, 2]
+    spectral_index = sources[:, 3]
+
+    ra_dec = np.deg2rad(ra_dec)
+
+    return ra_dec, flux, spectral_index
+
+
+def beams(nants):
+    """
+    Zernike polynomial beams with some randomly-chosen coefficients.
+    """
+    cfg_beam = dict(
+        ref_freq=1.0e8,
+        spectral_index=-0.6975,
+        beam_coeffs=[
```

Comment:
these one-per-line expansions by black on lists of floats are kind of annoying. You can avoid it by using `# nofmt` on the first line of the list.